src/gl/gl3device.cpp

```diff
@@ -25,20 +25,27 @@
 
 namespace rw {
 namespace gl3 {
-#ifndef LIBRW_SDL2
 struct DisplayMode
 {
+#ifdef LIBRW_SDL2
+	SDL_DisplayMode mode;
+#else
 	GLFWvidmode mode;
+#endif
 	int32 depth;
 	uint32 flags;
 };
-#endif
 
 struct GlGlobals
 {
 #ifdef LIBRW_SDL2
 	SDL_Window *window;
 	SDL_GLContext glcontext;
+
+	DisplayMode *modes;
```

Comment:
Perhaps re-order the struct members and only keep the SDL/GLFW members inside the `#ifdef`'ery?

Reply:
OK, done.